index.html

```diff
@@ -2395,11 +2449,14 @@ <h3>Algorithm</h3>
         <li>If <var>result</var> contains the <a>entry</a> <code>@value</code>:
           <ol>
             <li>The <var>result</var> must not contain any <a>entries</a> other than
-              <code>@value</code>, <code>@language</code>, <code>@type</code>,
-              and <code>@index</code>. It must not contain both a
-              <code>@language</code> <a>entry</a> and an <code>@type</code> <a>entry</a>.
-              Otherwise, an
-              <a data-link-for="JsonLdErrorCode">invalid value object</a>
+              <code class="changed">@direction</code>,
+              `@index`,
+              `@language`,
+              `@type`,
+              and `@value`.
+              It must not contain either
+              `@language` or `@direction` <a>entries</a> and an `@type` <a>entry</a>.
```

Comment:
I had to read this sentence several times to get it right (and I knew in advance what it was supposed to say...).

I suggest to rephrase it as follows (sorry, can not make a patch on multiple lines):

> It must not contain a `@type` entry if it contains either `@language` or `@direction` entries.